src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/Bindings/Web.cs

```diff
@@ -22,10 +24,13 @@ public class Web
         public static Hook<IActionResult> ActionResultHook;
 
         private readonly TestContext _context;
+        private static readonly Func<SpecifiedTimeProvider> _time = () => _timeProvider;
+        private static SpecifiedTimeProvider _timeProvider;
 
         public Web(TestContext context)
         {
             _context = context;
+            _timeProvider = _context.Time;
```

Comment:
This is unusual, a static value being set to a value from a non static instance. Does it get disposed correctly

Reply:
You're right, the `_timeProvider` of the last test context won't be garbage collected, but the test application is shutting down at that point anyway.